client/a8c-for-agencies/sections/overview/sidebar/contact-support/style.scss

```diff
@@ -0,0 +1,30 @@
+.overview__contact-support {
+	padding: 8px;
+	border-radius: 4px;
+
+	.foldable-card__header {
+		padding-left: 12px;
+		padding-right: 12px;
```

Comment:
[Non-blocker] We could [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) everywhere on this file since we can support RTL locales in Calypso.